Detectors/EMCAL/base/include/EMCALBase/ClusterFactory.h

```diff
@@ -311,6 +311,9 @@ class ClusterFactory
   /// \param  key: = 0(gamma, default); !=  0(electron)
   Double_t tMaxInCm(const Double_t e = 0.0, const int key = 0) const;
 
+  bool getLookUpInit() const { return mLookUpInit; }
+  void setLookUpInit(bool lookUpInit) { mLookUpInit = lookUpInit; }
```

Comment:
mLookupInit is a state variable, I would not expose it to the world.

Reply:
Done!